modules/thermo-cycler/thermo-cycler-arduino/thermo-cycler-arduino.ino

```diff
@@ -15,6 +15,8 @@ Adafruit_NeoPixel_ZeroDMA strip(NUM_PIXELS, NEO_PIN, NEO_RGBW);
 Adafruit_NeoPixel_ZeroDMA strip(NUM_PIXELS, NEO_PIN, NEO_GRB + NEO_KHZ800);
 #endif
 
+unsigned long timeStamp = 0;
```

Comment:
Should this be in here?

Reply:
I want to keep it until we finish changing the sample rate and adding timer interrupt. I want to monitor loop() duration for a little while